### DISCUSSION TITLE
An unexpected error occurred. Check application logs.

### DISCUSSION THREAD

**Discussion by jkill28** [mtalcott/google-photos-deduper#31] [Q&A] [answered]
Hi,
I have set up everything successfully and the app goes through my entire library (65000+ photos) without issues but at the end, I get the following error before I can start the last step to review and delete duplicates:
Whoops! An unexpected error occurred. Check application logs.
The Chrome extension shows the following error and Highlights the import function:
src/scripts/app_content.ts:3

**import** {
  DeletePhotoResultMessageType,
  HealthCheckMessageType,
  HealthCheckResultMessageType,
  StartDeletionTaskMessageType,
  StartDeletionTaskResultMessageType,
  StopDeletionTaskMessageType,
}

Any idea on what might be the cause ?

**Comment by romaan7** [comment]
I am getting this same issue while loading the chrome extenstion

**Comment by romaan7** [comment, marked as answer]
try loading the chrome extension from **chrome_extension\dist** insted of **chrome_extension\**

**Comment by jkill28** [comment]
thanks, it has gone further but now failing here:

`
[2024-01-07 05:19:34,797: ERROR/ForkPoolWorker-31] Task app.tasks.process_duplicates[6d8fefb2-7fe3-43bc-8d0c-e739d3a8880f] raised unexpected: SubtasksFailedError('1 of 814 subtasks failed. View log/celery_worker.log for more details. Restart to try again.')
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/celery/app/trace.py", line 477, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/src/app/app/__init__.py", line 25, in __call__
    return self.run(*args, **kwargs)
  File "/usr/src/app/app/tasks.py", line 111, in process_duplicates
    results = task_instance.run()
  File "/usr/src/app/app/lib/process_duplicates_task.py", line 92, in run
    self._await_subtask_completion()
  File "/usr/src/app/app/lib/process_duplicates_task.py", line 247, in _await_subtask_completion
    raise SubtasksFailedError(
app.lib.process_duplicates_task.SubtasksFailedError: 1 of 814 subtasks failed. View log/celery_worker.log for more details. Restart to try again.`

**Reply by mtalcott** [reply]
Can you share `log/celery_worker.log` (or a relevant excerpt) so I can see what the error is?

**Reply by mtalcott** [reply]
Fixed by https://github.com/mtalcott/google-photos-deduper/pull/36

**Comment by jkill28** [comment]
sure, see attached.
[celery_worker.log](https://github.com/mtalcott/google-photos-deduper/files/13883768/celery_worker.log)

**Reply by mtalcott** [reply]
Thanks for this! Helped me address in the right way. Should be fixed by https://github.com/mtalcott/google-photos-deduper/pull/36 now, if you pull from `main` branch.

**Comment by jkill28** [comment]
Brilliant, looking forward to test this out in the coming days.
Thanks!

**Comment by jkill28** [comment]
it' going very well now, in the process of deleting about 8000 duplicates !!
thank you so much for your time and effort, what a great tool :)

**Comment by jkill28** [comment]
ok, I completed the clean-up of all duplicates of my account, so good !
It crashed quite a lot of time, the Chrome tab was often crashing with "out of memory" which is rather surprising, my PC has 32 GB of RAM.
Anyway, not a problem, I restarted the process multiple times to get rid of them all.

Thanks a lof for your tool !

**Reply by mtalcott** [reply]
Interesting. Thanks for surfacing, I will take a look and see if I can figure out the out of memory errors as time permits.

Glad the tool is proving useful for you!
